mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraModSeqModule.java

```diff
@@ -46,6 +46,7 @@ public CassandraModSeqModule() {
                     .addPartitionKey(CassandraMessageModseqTable.MAILBOX_ID, timeuuid())
                     .addColumn(CassandraMessageModseqTable.NEXT_MODSEQ, bigint())
                     .withOptions()
+                    .comment("Holds and is used to generate MODSEQ. A monotic counter is implemented on top of this table.")
```

Comment:
No.

We also have a monotic counter implemented for UID. But both of these table serve a different purpose, hence I did not say just *monotic counter*.